ui/src/app/views/workspace-share/component.html

```diff
@@ -0,0 +1,32 @@
+<a style="display: block; padding: 0.4rem" routerLink="/..">&lt;&lt; Back to the workspace</a>
```

Comment:
I know we've touched on inline styles before -- I strongly prefer that all styles live in stylesheets. But maybe I shouldn't be in the business of pushing conventions at this point. (-: So feel free to ignore.

Some example implications that come to mind are CSS compilation (if we get around to switching to SCSS, we'll get validation of *.css but probably not *.html), easier refactoring (grep rem on *.css will find all the units to change, but not inline styles), reuse, and just finding style definitions (easier if I only have to look in one file).

Reply:
Agreed, I'll move all those inline styles to classes.